src/routes/(app)/add-ons/+page.svelte

```diff
@@ -10,7 +10,6 @@
   $: addons = data.addons;
   $: events = data.events;
   $: runs = data.runs;
-  $: query = ($page.url as URL).searchParams.get("query") ?? "";
```

Comment:
I think we want to be passing `query` through to the `AddOnBrowser`

Reply:
I think you're right